layers/meta-balena-genericx86/recipes-kernel/linux/balena-image-initramfs.bbappend

```diff
@@ -0,0 +1 @@
+PACKAGE_INSTALL_append_genericx86-64-ext = " initramfs-module-cryptsetup initramfs-module-kexec"
```

Comment:
Move it to meta-balena and make it conditional on SIGN_API.

Reply:
This should be in `recipes-core/images` and not `recipes-kernel/linux`. Anyway, it's now moved to meta-balena in https://github.com/balena-os/meta-balena/pull/2405